mne/viz/_3d.py

```diff
@@ -990,6 +1078,225 @@ def time_label(t):
+            ``lims`` : list | np.ndarray | tuple of float, 3 elements
+                Note: Only use this if 'colormap' is not 'mne'.
+                Left, middle, and right bound for colormap.
+            ``pos_lims`` : list | np.ndarray | tuple of float, 3 elements
+                Note: Only use this if 'colormap' is 'mne'.
+                Left, middle, and right bound for colormap. Positive values
+                will be mirrored directly across zero during colormap
+                construction to obtain negative control points.
+
+    size : float or pair of floats
+        The size of the window, in pixels. can be one number to specify
+        a square window, or the (width, height) of a rectangular window.
+    background : matplotlib color
+        Color of the background of the display window.
+    foreground : matplotlib color
+        Color of the foreground of the display window.
+
+    Returns
+    -------
+    brain : Brain
+        A instance of surfer.viz.Brain from PySurfer.
+
+    Note
+    ----
+    PySurfer currently needs the SUBJECTS_DIR environment variable, which will
+    automatically be set by this function. Plotting multiple SourceEstimates
+    with different values for subjects_dir will cause PySurfer to use the wrong
+    FreeSurfer surfaces when using methods of the returned Brain object. It is
+    therefore recommended to set the SUBJECTS_DIR environment variable or
+    always use the same value for subjects_dir (within the same Python
+    session).
+    """
+    import surfer
+    from surfer import Brain
+    import mayavi
+
+    # import here to avoid circular import problem
+    from ..source_estimate import VectorSourceEstimate
+
+    surfer_version = LooseVersion(surfer.__version__)
+    v06 = LooseVersion('0.6')
+    if surfer_version < v06:
+        raise ImportError("This function requires PySurfer 0.6 (you are "
+                          "running version %s). You can update PySurfer "
+                          "using:\n\n    $ pip install -U pysurfer" %
+                          surfer.__version__)
+
+    if not isinstance(stc, VectorSourceEstimate):
+        raise ValueError('stc has to be a vector source estimate')
+
+    if hemi not in ['lh', 'rh', 'split', 'both']:
+        raise ValueError('hemi has to be either "lh", "rh", "split", '
+                         'or "both"')
+
+    # check `figure` parameter (This will be performed by PySurfer > 0.6)
+    if figure is not None:
+        if isinstance(figure, int):
+            # use figure with specified id
+            size_ = size if isinstance(size, (tuple, list)) else (size, size)
+            figure = [mayavi.mlab.figure(figure, size=size_)]
+        elif not isinstance(figure, (list, tuple)):
+            figure = [figure]
+        if not all(isinstance(f, mayavi.core.scene.Scene) for f in figure):
+            raise TypeError('figure must be a mayavi scene or list of scenes')
```

Comment:
You shouldn't need to replicate this stuff. Let's just require 0.7, it's pure Python and pip installable